#+BEGIN_HTML
---
layout: post
class: post-template
title: "Predict hosts and attachment sites of lysogenic phages"
author: thackl
cover: img/phage-host-pred-with-phage.png
---
#+END_HTML

Thanks to new sequencing and assembly strategies, new viral genomes are
currently uncovered on what feels like a daily basis. Virus-host relationships,
however, often remain unclear because most genomes are obtained from community
samples rather than cultures. Here, I describe a way to link phages with a
lysogenic lifestyle to their respective hosts using only sequence information.

Lysogenic phages - viruses that can integrate into their host's genome - often
share a short sequence with their host that serves as attachment site during the
integration. This roughly 40bp long motif typically is part of one of the host's
tRNA genes. By analyzing short matches between phage contigs and tRNAs of
potential hosts (sympatric microbes), it is possible to predict phage attachment
sites and putative host, i.e. bacteria carrying the respective tRNA.

To facilitate this kind of analysis, I wrote [[https://github.com/thackl/host-phage-tRNA-att-finder%5D%5Bhost-phage-tRNA-att-finder][host-phage-tRNA-att-finder]], a small
toolkit to run the following workflow:

1. Predict host tRNAs in putative host contigs using [[https://dx.doi.org/10.1093%252Fnar%252Fgkh152][ARAGORN]] *or*
Supply a set of known tRNAs.
2. Blast tRNAs against the phage contigs to identify high-identity short matches
   (>80% identity, ~20-40bp) with match/gap cost favoring these kinds of hits.
3. Predict Integrase genes on phage contigs as sanity check - lysogenic phages
   with an attachment site should also carry an integrase gene.
3. Aggregate and filter the results for spurious or ambiguous matches.
4. Plot the distribution of filtered hits across different taxonomic classes.

