<p style="text-align: right; margin-top: -40px;"><small>Last updated <i>March 04, 2019</i>. Pulled from my <a href="https://scholar.google.com/citations?hl=en&user=b8bWNkUAAAAJ">Google Scholar profile</a></small></p>
<h3>2018</h3>
<table class="publication-table" border="10px solid blue" cellspacing="0" cellpadding="6" rules="", frame=""><tbody>
<tr><td width="100%">PM Berube, SJ Biller, <b>T Hackl</b>, SL Hogle, BM Satinsky, JW Becker et al. (2018) <a href="https://scholar.google.com/scholar?oi=bibs&cluster=5858880749316430558&btnI=1&hl=en">Single cell genomes of Prochlorococcus, Synechococcus, and sympatric microbes from diverse marine environments</a>, Scientific data, 5</td></tr>
<tr><td width="100%">SJ Biller, PM Berube, K Dooley, M Williams, BM Satinsky, <b>T Hackl</b> et al. (2018) <a href="https://scholar.google.com/scholar?oi=bibs&cluster=17343079802559454777&btnI=1&hl=en">Marine microbial metagenomes sampled across space and time</a>, Scientific data, 5, 180176</td></tr>
<tr><td width="100%">JH Grau, <b>T Hackl</b>, KP Koepfli, M Hofreiter (2018) <a href="https://scholar.google.com/scholar?oi=bibs&cluster=12617815340339850483&btnI=1&hl=en">Improving draft genome contiguity with reference-derived in silico mate-pair libraries</a>, GigaScience, 7 (5), giy029</td></tr>
<tr><td width="100%">N Terhoeven, J Schultz, <b>T Hackl</b> (2018) <a href="https://scholar.google.com/scholar?oi=bibs&cluster=NA&btnI=1&hl=en">reper: Genome-wide identification, classification and quantification of repetitive elements without an assembled genome</a>, Journal of Open Source Software, </td></tr>
<tr><td width="100%">MJ Ankenbrand, S Pfaff, N Terhoeven, M Qureischi, M G&uuml;ndel, CL Wei&szlig; et al. (2018) <a href="https://scholar.google.com/scholar?oi=bibs&cluster=NA&btnI=1&hl=en">chloroExtractor: extraction and assembly of the chloroplast genome from whole genome shotgun data</a>, The Journal of Open Source Software, 3</td></tr>
</tbody></table>
<h3>2017</h3>
<table class="publication-table" border="10px solid blue" cellspacing="0" cellpadding="6" rules="", frame=""><tbody>
<tr><td width="100%">BM Slaby, <b>T Hackl</b>, H Horn, K Bayer, U Hentschel (2017) <a href="https://scholar.google.com/scholar?oi=bibs&cluster=5037588228984496091&btnI=1&hl=en">Metagenomic binning of a marine sponge microbiome reveals unity in defense but metabolic specialization</a>, The ISME journal, 11 (11), 2465</td></tr>
<tr><td width="100%">MJ Ankenbrand, S Hohlfeld, <b>T Hackl</b>, F F&ouml;rster (2017) <a href="https://scholar.google.com/scholar?oi=bibs&cluster=15319190401677747655&btnI=1&hl=en">AliTV—interactive visualization of whole genome comparisons</a>, PeerJ Computer Science, 3, e116</td></tr>
</tbody></table>
<h3>2016</h3>
<table class="publication-table" border="10px solid blue" cellspacing="0" cellpadding="6" rules="", frame=""><tbody>
<tr><td width="100%">MG Fischer, <b>T Hackl</b> (2016) <a href="https://scholar.google.com/scholar?oi=bibs&cluster=11951868283239200495,8264985959565825189&btnI=1&hl=en">Host genome integration and giant virus-induced reactivation of the virophage mavirus</a>, Nature, 540 (7632), 288</td></tr>
<tr><td width="100%"><b>T Hackl</b> (2016) <a href="https://scholar.google.com/scholar?oi=bibs&cluster=NA&btnI=1&hl=en">A draft genome for the Venus flytrap, Dionaea muscipula: Evaluation of assembly strategies for a complex Genome–Development of novel approaches and bioinformatics solutions</a>, , </td></tr>
</tbody></table>
<h3>2014</h3>
<table class="publication-table" border="10px solid blue" cellspacing="0" cellpadding="6" rules="", frame=""><tbody>
<tr><td width="100%"><b>T Hackl</b>, R Hedrich, J Schultz, F F&ouml;rster (2014) <a href="https://scholar.google.com/scholar?oi=bibs&cluster=2799924011968743286&btnI=1&hl=en">proovread             : large-scale high-accuracy PacBio correction through iterative short read consensus</a>, Bioinformatics, 30 (21), 3004-3011</td></tr>
</tbody></table>
<h3>2012</h3>
<table class="publication-table" border="10px solid blue" cellspacing="0" cellpadding="6" rules="", frame=""><tbody>
<tr><td width="100%">C Koetschan, <b>T Hackl</b>, T M&uuml;ller, M Wolf, F F&ouml;rster, J Schultz (2012) <a href="https://scholar.google.com/scholar?oi=bibs&cluster=17348206402672079086&btnI=1&hl=en">ITS2 database IV: interactive taxon sampling for internal transcribed spacer 2 based phylogenies</a>, Molecular Phylogenetics and Evolution, 63 (3), 585-588</td></tr>
<tr><td width="100%">B Merget, C Koetschan, <b>T Hackl</b>, F F&ouml;rster, T Dandekar, T M&uuml;ller et al. (2012) <a href="https://scholar.google.com/scholar?oi=bibs&cluster=2897669303996040886&btnI=1&hl=en">The ITS2 database</a>, Journal of visualized experiments: JoVE, </td></tr>
</tbody></table>
